README.md

```diff
@@ -497,6 +493,8 @@ For information about the governance of the Node.js project, see
   **Andreas Madsen** <<amwebdk@gmail.com>> (he/him)
 * [aqrln](https://github.com/aqrln) -
   **Alexey Orlenko** <<eaglexrlnk@gmail.com>> (he/him)
+* [atlowChemi](https://github.com/atlowChemi) -
```

Comment:
I think another case of mismatch with the mail ids? cc @atlowChemi

Reply:
@debadree25 I rebased without the old commit 🙂

Reply:
Ahh got it!